README.md

```diff
@@ -67,65 +33,80 @@ A React component class that injects subscriptions and actions into your compone
 
 #### Remarks
 
-* It needs to be invoked two times. The first time with its arguments described above, and a second time, with the component: `connect(mapPropsToSubscriptions, mapFirebaseToProps, mergeProps)(MyComponent)`.
+* It needs to be invoked two times. The first time with its arguments described above, and a second time, with the component: `connect(mapFirebaseToProps)(MyComponent)`.
```

Comment:
What? Does this effect the developer or component in a way we should be vary about?